public/electron.js

```diff
@@ -2,7 +2,6 @@
 if(require('electron-squirrel-startup')) return 
 
 // Modules to control application life and create native browser window
-const imageKeyboardResponse = require('@jspsych/plugin-html-keyboard-response')
```

Comment:
I missed this in my last PR to upgrade to jspsych 7, which I didn't catch until I started testing with electron.  jspsych 7 will only run in the browser, so we can't `require()` it here on the server-side.

I'm hoping `args.trial_type` below can happily be a string, although I don't quite know how these `args` are used.